src/main/java/com/amihaiemil/camel/RtYamlLine.java

```diff
@@ -82,4 +82,17 @@ public int indentation() {
     public String toString() {
         return this.value;
     }
+
+    @Override
+    public int compareTo(final YamlLine other) {    
```

Comment:
This is the only place where the method is actually implemented, since ``RtYamlLine`` is the base implementation of ``YamlLine``. 

Looks good, but instead of ``toString()``, use method ``trimmed()``, because we don't want spaces at the beginning to affect the alphabetical order of the lines (also, I think I'm going to modify ``trimmed()`` a little, to also cut any ``-`` from the beginning of the line)